easybuild/easyconfigs/b/BLOCK/BLOCK-1.5.3-intel-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'MakeCp'
+
+name = 'BLOCK'
+version = '1.5.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://sanshar.github.io/Block/'
+description = """BLOCK implements the density matrix renormalization group (DMRG) algorithm for quantum chemistry."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'cstd': 'c++11'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.sunqm.net/pyscf/files/src/']
```

Comment:
It is necessary to add the checksums of all sources and files. You can do so with the command `eb --inject-checksums`